pkg/controllers/machine/lifecycle/metrics.go

```diff
@@ -0,0 +1,46 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lifecycle
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/aws/karpenter-core/pkg/metrics"
+)
+
+var (
+	MachinesRegisteredCounter = prometheus.NewCounterVec(
```

Comment:
Any reason why we can't put these two with the other metrics in pkg/metrics/metrics.go?